mkdocs/docs/getting-started/your-first-experiment.md

```diff
@@ -4,38 +4,36 @@ template: main.html
 
 # Your First Experiment
```

Comment:
we changed this to "quick start" in the README. Do we want the same here?

Reply:
Let us keep it as is for now. I think `your second experiment`  may be desirable in the getting started section. This doesn't look like a sensible option in the GitHub README though, which is the reason for the distinction.